src/main/java/com/hortonworks/spark/sql/hive/llap/HWConf.java

```diff
@@ -0,0 +1,45 @@
+package com.hortonworks.spark.sql.hive.llap;
+
+import java.util.Optional;
+
+enum HWConf {
+
+  USER("user.name", warehouseKey("user.name"), ""),
+  PASSWORD("password", warehouseKey("password"), ""),
+  HS2_URL("hs2.url", warehouseKey("hs2.url"), "jdbc:hive2://localhost:10084"),
```

Comment:
Looks like you were probably using this for testing, but a better default port might be 10500 (default port for HS2 with LLAP)

Reply:
Addressed in https://github.com/hortonworks-spark/spark-llap/pull/211